libraries/libmagic/make.sh

```diff
@@ -0,0 +1,76 @@
+PKGURL=https://github.com/threatstack/libmagic/archive/5.18.tar.gz
+PKGHASH=ce734a1cc24bddbfd8b21f4ffdfe721ab74eeed9
+
+package_download $PKGURL $PKGHASH
+package_patch
+
+EXTRACONF=
+
+if [ "$SYS_PLATFORM" != "$SYS_HOSTPLATFORM" ]; then
+    EXTRACONF="$EXTRACONF --host=$SYS_ARCH"
+fi
+
+# pretend libgnurx exists by masking libpcre
+
+test -L $SYS_PREFIX/include/regex.h || ln -s pcreposix.h $SYS_PREFIX/include/regex.h
+
+# Mabye instead of fooling the package, it should be patched
+rmifexists $SYS_PREFIX/lib/libgnurx.a
+rmifexists $SYS_PREFIX/lib/libgnurx
+mkdir $SYS_PREFIX/lib/libgnurx
+( cd $SYS_PREFIX/lib/libgnurx
+  mkdir a
+  cd a
+  $SYS_AR -x ../../libpcre.a
+  cd ..
+  mkdir b
+  cd b
+  $SYS_AR -x ../../libpcreposix.a
+  cd ..
+  $SYS_AR rc ../libgnurx.a a/*.o b/*.o
+)
+
+rm -r $SYS_PREFIX/lib/libgnurx
+
+if [ `file --version|head -1` != file-5.18 ]; then
+    if [ ! -f  $SYS_HOSTPREFIX/bin/file ]; then
+	./configure --enable-static --disable-shared 'CFLAGS=-fPIC -O3 -DPCRE_STATIC'
+	package_make
+	cp src/file $SYS_HOSTPREFIX/bin || exit 1
+	package_cleanup
+	package_download $PKGURL $PKGHASH
```

Comment:
shouldn't it already be downloaded by this time (as in line 4)?

Reply:
Yes it should reading `package.sh` again I now see that this `package_download` might be better replaced by a `package_unpack`.  I just was not aware of the latter being available as a separate call.